databuilder/databuilder/extractor/neo4j_extractor.py

```diff
@@ -25,15 +26,18 @@ class Neo4jExtractor(Extractor):
     MODEL_CLASS_CONFIG_KEY = 'model_class'
     NEO4J_AUTH_USER = 'neo4j_auth_user'
     NEO4J_AUTH_PW = 'neo4j_auth_pw'
+    NEO4J_DATABASE_NAME = 'neo4j_database'
     NEO4J_MAX_CONN_LIFE_TIME_SEC = 'neo4j_max_conn_life_time_sec'
     NEO4J_ENCRYPTED = 'neo4j_encrypted'
     """NEO4J_ENCRYPTED is a boolean indicating whether to use SSL/TLS when connecting."""
     NEO4J_VALIDATE_SSL = 'neo4j_validate_ssl'
     """NEO4J_VALIDATE_SSL is a boolean indicating whether to validate the server's SSL/TLS cert against system CAs."""
+    NEO4J_DRIVER = 'neo4j_driver'
 
-    DEFAULT_CONFIG = ConfigFactory.from_dict({NEO4J_MAX_CONN_LIFE_TIME_SEC: 50,
-                                              NEO4J_ENCRYPTED: True,
```

Comment:
pretty sure this is a break?

Reply:
encrypted was true by default in the old client. in the new client, it is off by default. One thing I am not clear about is what specifying it explicitly as "true" in the new client will do. Ideally i'd just copy the behavior of the old.

Reply:
the motivation is we don't want folks who were used to getting encryption to suddenly not get it. We'd want (assuming the server neo4j is new enough) for the behavior people get to be unchanged if they don't do anything. At least when it comes to really significant things like encryption. If that's not possible, they should get an error asking them to change their parameters.